encrypted-media/drm-not-callable-after-createsession.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <meta charset=utf-8>
+    <meta name="timeout" content="long">
```

Comment:
I don't think this test need needs `drm-messagehandler.js`. `not-callable-after-createsession.js` does not handle messages.

We should remove it from `clearkey-` too.